templates/metrics/css/animated.html

```diff
@@ -30,7 +30,9 @@
   <div>
     <h3>About this data</h3>
     <p class="description">
-    We've been using Chrome's <a href="https://cs.chromium.org/chromium/src/tools/metrics/histograms/enums.xml" target="_blank">anonymous usage statistics</a> to count the CSS properties which are animated. A property that is animated multiple times during a single page view is counted each time it animates. <b>Percentages on this page indicate the amount of times (as the fraction of all animated properties) each property is used in a Transition or Animation</b>. Data is ~24 hrs old and reflects usage across all channels and platforms.</p>
+    We've been using Chrome's <a href="https://cs.chromium.org/chromium/src/tools/metrics/histograms/enums.xml" target="_blank">anonymous usage statistics</a> to count the CSS properties which are animated.
+    A property that is animated multiple times during a single page view is counted each time it animates.
```

Comment:
The above line is no longer true, just delete it?  The statement below accurately reflects how it works.

Reply:
Do you mean just keep this sentence:

"Perrcentages on this page indicate ...

Reply:
Yes.  Remove line 34 (as it's contradictory to line 35 IMHO).

Reply:
done